prog/vm/nexus.rb

```diff
@@ -594,6 +595,8 @@ def write_params_json
         used_hugepages_1g: Sequel[:used_hugepages_1g] - vm.mem_gib
       )
 
+      PciDevice.dataset.where(vm_id: vm.id).update(vm_id: nil)
```

Comment:
Is it possible to use `association_dependencies` plugin here?

https://github.com/ubicloud/ubicloud/blob/9efe7f8c3889e3f0f6f0cad1a2997005c124d805/model/vm.rb#L16

Reply:
we don't destroy the pci_device, just update it, in contrast to these entities. the update we do to pci devices is similar to what we do to the vm_host or storage_devices, where we don't use `association_dependencies`, either.

Reply:
`association_dependencies` also has `:nullify` option instead of `:destroy`. But yes we didn't use `:nullify` before.
https://sequel.jeremyevans.net/rdoc-plugins/classes/Sequel/Plugins/AssociationDependencies.html